DeKast/src/app/login/login.component.scss

```diff
@@ -0,0 +1,230 @@
+
+
+section.login{
+    display: flex;
+    height: 100vh;
+    overflow-x: hidden;
+    @media (max-width: 750px) {
+        height: fit-content;
+        flex-direction: column;
+    }
+    .form-side{
+        width: 35%;
+        min-width: 400px;
+        padding: 70px 120px;
+        background-color: var(--color1);
+        box-shadow: 5px 10px 10px rgba(0, 0, 0, 0.3);
+        z-index: 5;
+
+        @media (max-width: 1400px) {
+            width: 60%;
+            padding: 50px 90px;
+        }
+        @media (max-width: 1200px) {
+            padding: 50px;
+        }
+        @media (max-width: 1000px) {
+            width: 70%;
+        }
+        @media (max-width: 750px) {
+            width: auto;
+            min-width: auto;
+            height: fit-content;

```

Comment:
zo veel media queries zijn echt nooit nodig moeten zijn. voor nu laat het maar, maar dit is echt overkill.